src/pudl/transform/ferc1.py

```diff
@@ -2584,6 +2585,26 @@ def spot_fix_rows(self, df: pd.DataFrame) -> pd.DataFrame:
         return df
 
 
+class TransmissionFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """A table transformer specific to the :ref:`transmission_ferc1` table."""
+
+    table_id: Ferc1TableId = Ferc1TableId.TRANSMISSION_FERC1
+
+    @cache_df(key="main")
+    def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Table specific transforms for plants_small_ferc1.
+
+        Params:
+            df: Pre-processed, concatenated XBRL and DBF data.
+
+        Returns:
+            A single transformed table concatenating multiple years of cleaned data
+            derived from the raw DBF and/or XBRL inputs.
+        """
+        df = self.replace_with_na(df).pipe(self.drop_invalid_rows)
```

Comment:
is there anything else from the `Ferc1AbstractTableTransformer.transform_main` that you might want for this table? you can always do `super().transform_main(df).pipe(self.replace_with_na)` if you want to tack something onto the end of the standard method. most of the default steps in `transform_main` do nothing if there are no defined params for that step so even if there are some that you don't need that's fine. I think it's good to add things into the default if possible. Like maybe even add this `replace_with_na` into the default `transform_main` which will  do nothing to all of the other tables that don't have a param for it..

Reply:
Hmm ok I guess I wasn't really doing this with the small plants table because the order of opperations was different (i.e., we needed to clean the columns before categorizing them.)